### PR TITLE
adding region

### PR DESCRIPTION
**Pull request by cdwilhelm**

### CODE REVIEW

**Comment by cdwilhelm** [comment]
@rshade @stefhen  peer review please?

File: files/default/ros.rb [hunk 1]
```diff
@@ -88,7 +95,7 @@ class MyCLI
     :google_storage_secret_access_key => @cli.config[:secret_key]
   }
 end
-
+puts "################ @credentials #{@credentials}"
```
Comment (stefhen) [inline files/default/ros.rb]: Do you want to echo credentials here?

File: metadata.rb [hunk 2]
```diff
@@ -56,3 +56,10 @@
   :type         => 'string',
   :recipes      => ['rsc_ros::download'],
   :required     => 'required'
+
+  attribute 'rsc_ros/region',
+    :display_name => 'Region',
+    :description  => 'Cloud region',
+    :type         => 'string',
+    :recipes      => ['rsc_ros::download'],
```
Comment (stefhen) [inline metadata.rb]: This should also include `rsc_ros::upload`, as it looks like region is being ref'd in the `upload.rb` script too.

**Comment by rshade** [comment]
:+1:

**Pull request merged**